downstream/modules/platform/con-operator-upgrade-prereq.adoc

```diff
@@ -0,0 +1,11 @@
+[id="operator-upgrade-prereq_{context}"]
+
+= Prerequisites
+
+
+[role="_abstract"]
+To upgrade to a newer version of {OperatorPlatform}, it is recommended that you do the following:
+
+* Create AutomationControllerBackup and AutomationHubBackup objects.
+//See (Backup and Restore) for information on creating backup objects. [add link to new backup and restore doc when complete]
```

Comment:
Will these links be upgraded before this peer review has progressed to QE?

Reply:
The backup and restore is for V2.3 so this information should be added for v2.3 and not this 2.2 version.